### PR TITLE
fix(core/entityTag): Clone more fields in UI when editing an entity tag

### PR DESCRIPTION
**Pull request by christopherthielen**
@anotherchrisberry ptal

### CODE REVIEW

**Comment by christopherthielen** [comment]
Editing an alert decategorizes it (and loses the imageId and any other fields)

File: app/scripts/modules/core/src/delivery/details/singleExecutionDetails.component.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { IScope, module } from 'angular';
 import { StateService } from '@uirouter/core';
-import { Subscription } from 'rxjs/Subscription';
+import { Subscription } from 'rxjs';
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/delivery/details/singleExecutionDetails.component.ts]: this is naturally my favorite part of the PR

**Comment by anotherchrisberry** [comment]
LGTM

File: app/scripts/modules/core/src/delivery/details/singleExecutionDetails.component.ts (hunk 1 above)
Comment (christopherthielen) [inline app/scripts/modules/core/src/delivery/details/singleExecutionDetails.component.ts]: this makes me sad

Comment (anotherchrisberry) [inline app/scripts/modules/core/src/delivery/details/singleExecutionDetails.component.ts]: how do you think it makes me feel

**Pull request merged**